python_ta/checkers/forbidden_io_function_checker.py

```diff
@@ -21,8 +24,8 @@ class IOFunctionChecker(BaseChecker):
         "E9998": (
             "Used input/output function %s",
             "forbidden-IO-function",
-            'Used when you use the I/O functions "print", "open" or "input". These '
-            "functions should not be used except where specified by your instructor.",
+            'Used when you use the I/O functions "print", "open" or "input" or other config specified forbidden '
```

Comment:
"config specified" -> "config-specified"

Remove the "/methods" (it's okay to just say "functions", as all methods are functions)